22 спринт,попытка "1"

Gammius opened the pull request.

Gammius committed:
22 спринт,попытка "1"

kosstyango commented:
⚠️ Критические замечания:
Созданные плейлисты не сразу появляются в списке на экране "Плейлист" (см.видео-скрин)
https://github.com/user-attachments/assets/0749a1a2-7862-4392-b6c8-d82599296d87

Окно добавления треков исчезает при попытке повторно добавить трек в плейлист, где такой трек уже есть. (см.видео-скрин)
https://github.com/user-attachments/assets/e497c7e4-c0f8-42b2-8c60-cd9380933994

🍏 Рекомендации, не влияющие на результат проверки работы:
Слово "трек/трека/треков" написано с ошибкой: "трэк/трэка/трэков"
Нет тоста (и других сообщений) при успешном создании нового плейлиста

Gammius committed:
22 спринт,попытка "1" исправил:
1.при создании плейлиста обновляется список плейлистов в BottomSheet
2.при добавлении ,если трек уже есть в плейлисте, то BottomSheet не скрывается
3.Исправил Toast, при создании плейлиста
4.трЭки -> трЕки
